core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts

```diff
@@ -72,6 +77,14 @@ export class DotFavoritePageComponent implements OnInit, AfterViewInit, OnDestro
                 this.ref.close(true);
             }
         });
+
+        this.store.actionState$
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((actionState: DotFavoritePageActionState) => {
+                if (actionState === DotFavoritePageActionState.SAVED) {
```

Comment:
I think this can be a `filter` from rxjs.

Reply:
mmm, I get your idea, but later when I implement the actionState of `DELETED` I will need to add more code, so correct me if I'm wrong, but I think a filter won't work